ConwaysGameOfLife.Tests/CellShould.cs

```diff
@@ -42,16 +42,44 @@ public void WhenInitialized_SetLocation(int x, int y)
         }
 
         [Theory]
-        [InlineData(CellState.Dead, 0, 1, CellState.Dead, 0, 1, true)]
-        [InlineData(CellState.Dead, 0, 1, CellState.Alive, 0, 1, false)]
-        public void GivenTwoCells_CellStateAndLocationAreEqual(CellState state1, int x1, int y1,  CellState state2, int x2, int y2, bool expected) 
+        [InlineData(CellState.Dead, 0, 1, CellState.Dead, 0, 1)]
+        [InlineData(CellState.Alive, 0, 1, CellState.Alive, 0, 1)]
+        public void GivenTwoCells_CellStateAndLocationAreEqual(CellState state1, int x1, int y1,  CellState state2, int x2, int y2) 
         {
             var one = new Cell(state1, x1, y1);
             var two = new Cell(state2, x2, y2);

```

Comment:
worth having cell in the names perhaps
```suggestion
            var cell1 = new Cell(state1, x1, y1);
            var cell2 = new Cell(state2, x2, y2);
```